guides/common/assembly_converting-a-host-with-convert2rhel.adoc

```diff
@@ -0,0 +1,58 @@
+
+Use the global registration template to register and subscribe your system before the conversion.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html/managing_hosts/registering_hosts#registering-a-host-to-project-using-the-global-registration-template_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in the _Managing Hosts_ guide.
+
+To convert hosts to {RHEL} with convert2RHEL, you must complete the following procedures:
+
+. xref:installing-convert2rhel_{context}[].
+. xref:creating-a-host-activation-key_{context}[].
+. xref:preparing-the-host-group_{context}[].
+. xref:registering-hosts-for-conversion_{context}[].
+. xref:converting-hosts-with-convert2rhel_{context}[]
+
+.Prerequisites
+* You must have completed the steps listed in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/converting_from_an_rpm-based_linux_distribution_to_rhel/index#proc_preparing-for-a-rhel-conversion_converting-from-a-linux-distribution-to-rhel[Preparing for a RHEL conversion].
+* If you are providing the organization ID and activation key to register and subscribe your system, you must have created an activation key in {ProjectName}.
+ifdef::satellite[]
+For {Project} you can also create an activation key in {Team} Subscription Manager.
+endif::[]
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.9/html/content_management_guide/managing_activation_keys[Managing Activation Keys] in this guide, and https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html-single/using_red_hat_subscription_management/index#activation_keys_con[Understanding activation keys] in {Team} Subscription Manager documentation.
+* Ensure you have a subscription manifest uploaded to your Satellite and that there are sufficient {RHEL} subscriptions for the conversion.
+* If you are providing the organization ID and activation key to register and subscribe your system, you must have created an activation key in {ProjectName} or {Team} Subscription Manager.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html/content_management_guide/managing_activation_keys[Managing Activation Keys] in this guide, and https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html-single/using_red_hat_subscription_management/index#activation_keys_con[Understanding activation keys] in {Team} Subscription Manager documentation.
+* Ensure you have a subscription manifest uploaded to your {Project} and that there are sufficient {RHEL} subscriptions for the conversion.
+
+[NOTE]
+====
+If you are accessing {RHEL} packages using {ProjectServer}, you must use the organization ID and activation key to run the `convert2rhel` command.
+====
```

Comment:
If you can avoid NOTE, avoid note markup. (Brno as well)

Reply:
Fixed